pkg/api/repository_parameters.go

```diff
@@ -26,9 +28,10 @@ type GenericAttributeValidationResponse struct {
 }
 
 type UrlValidationResponse struct {
-	Skipped         bool   `json:"skipped"`          // Skipped if the URL is not passed in for validation
-	Valid           bool   `json:"valid"`            // Valid if not skipped and provided attribute is valid to be saved
-	Error           string `json:"error"`            // Error message if the attribute is not valid
-	HTTPCode        int    `json:"http_code"`        // If the metadata cannot be fetched successfully, the http code that is returned if the http request was completed
-	MetadataPresent bool   `json:"metadata_present"` // True if the metadata can be fetched successfully
+	Skipped                  bool   `json:"skipped"`                    // Skipped if the URL is not passed in for validation
+	Valid                    bool   `json:"valid"`                      // Valid if not skipped and provided attribute is valid to be saved
```

Comment:
```suggestion
	Valid                    bool   `json:"valid"`                      // Valid if not skipped and the provided attribute is valid
```
"to be saved" seems redundant